f2/package.json

```diff
@@ -70,6 +70,7 @@
     "babel-core": "^7.0.0-bridge.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.15.1",
+    "husky": "^4.0.10",
```

Comment:
Where does this come from? @sastels

Reply:
It's something I was going to look at last week. A package to enforce prettier requirements. Shouldn't have any affect on prod code. 👍